tests/env_freeipa_facts.yml

```diff
@@ -15,12 +15,12 @@
 - name: Verify if host is an IPA server or client.
   shell:
     cmd: |
-      echo SomeADMINpassword | kinit -c {{ KRB5CCNAME }} admin
-      RESULT=$(KRB5CCNAME={{ KRB5CCNAME }} ipa server-show `hostname` && echo SERVER || echo CLIENT)
-      kdestroy -A -c {{ KRB5CCNAME }}
+      echo SomeADMINpassword | kinit -c {{ krb5ccname }} admin
+      RESULT=$(krb5ccname={{ krb5ccname }} ipa server-show `hostname` && echo SERVER || echo CLIENT)
```

Comment:
The shell variable should still be KRB5CCNAME (uppercase). To avoid confusion, could the "lowercase version" be changed to something like krb5_cache_name?